src/cli-client/internals/cli/cli_upload.go

```diff
@@ -0,0 +1,121 @@
+package cli
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/canonical/oci-factory/cli-client/internals/client"
+	"github.com/canonical/oci-factory/cli-client/internals/logger"
+	"github.com/canonical/oci-factory/cli-client/internals/trigger"
+)
+
+type UploadRelease struct {
+	Track     string
+	Risks     []string
+	EndOfLife string
+}
+
+type CmdUpload struct {
+	UploadRelease []string `long:"release" description:"Release images to container registries.\nSyntax: --release tracks=<release track>,risks=<risk1>[,<risk2>...],eol=yyyy-mm-dd"`
```

Comment:
@cjdcordeiro Since we are only releasing to one track with one call of `--release`, should we change it to "track"? (Currently in our spec it's still "tracks".)

Reply:
It should be `track`. Which means we might end up some repetition like `--release track=foo,risks=edge,beta,eol=01-2025 --release track=foo-alias,risks=edge,beta,eol=01-2025`. But that's ok for now. We can later introduce a new option `track-aliases` such that one could have `--release track=foo,risks=edge,beta,eol=01-2025,track-aliases=foo-alias,foo2...`

Reply:
Refactored and noted in ROCKS-1233.